runtime/flashbox/src/lib.rs

```diff
@@ -1284,13 +1284,21 @@ impl pallet_stream_payment::TimeProvider<TimeUnit, Balance> for TimeProvider {
 
 type StreamId = u64;
 
+parameter_types! {
+    // 1 entry, storing 173 bytes on-chain
+    pub const OpenStreamHoldAmount: Balance = currency::deposit(1, 173);
```

Comment:
Just curious, how did you get this value?

Reply:
Did a raw query of a Stream that had a `pending_request` and used that size